build/version.go

```diff
@@ -44,7 +44,7 @@ const (
 	AppMinor uint = 14
 
 	// AppPatch defines the application patch for this binary.
-	AppPatch uint = 1
+	AppPatch uint = 99
```

Comment:
I'm not sure I follow why we need to bump to 99?

Reply:
We usually do that directly after releasing a new major version because we create branches for the minor versions. Without this, master is still at `v0.14.1-beta` while certain applications might already require `v0.14.2-beta` as the minimum version (Loop for example).